src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public enum Color {
+    red,
+    orange,
+    yellow,
+    green,
+    blue,
+    purple
```

Comment:
Enum values should be written in uppercase letters. For example, 'RED' instead of 'red'. Please correct the naming for all enum constants.

Reply:
not fixed

Reply:
```suggestion
    RED,
    ORANGE,
    YELOW,
    green,
    blue,
    purple
```